# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/mips64/Architecture-mips64.h
# Commit: a0e51f584680
# Full Hash: a0e51f584680a017fc4d8124c58a51f3caadfade
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2021-09-16 09:43:03
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Introduce indirect stubs to optimize call_indirect. r=lth
#   
#   This patch introduces indirect stubs.
#   An indirect stub is a stub that takes care of any switching activities needed for call_indirect.
#   Before this patch, we have to conservatively assume that any call_indirect's target can be from a foreign instance,
# ==============================================================================

diff -r ece3dcc343a2 -r a0e51f584680 js/src/jit/mips64/Architecture-mips64.h
--- a/js/src/jit/mips64/Architecture-mips64.h	Thu Sep 16 04:38:26 2021 +0000
+++ b/js/src/jit/mips64/Architecture-mips64.h	Thu Sep 16 05:09:24 2021 +0000
@@ -21,6 +21,7 @@
 
 // Shadow stack space is not required on MIPS64.
 static const uint32_t ShadowStackSpace = 0;
+static const uint32_t SizeOfReturnAddressAfterCall = 0;
 
 // MIPS64 have 64 bit floating-point coprocessor. There are 32 double
 // precision register which can also be used as single precision registers.